### PR TITLE
Finish the job of listPairedDevices for windows

### PR DESCRIPTION
**Pull request by checksummaster**
The structure contain all real service name and port.
Using listPairedDevices on mac and now on windows let us choose the service we want to connect.

### CODE REVIEW

**Comment by eelcocramer** [comment]
Thank you for contributing with this PR! Will take a look at it soon.

**Comment by barchard** [comment]
hi, just curious if there was any progress on this PR? i'm integrating this into an electron app and interested in testing/evaluating these changes on Windows. Thanks.

**Comment by eelcocramer** [comment]
Thanks for reminding me. 

I just released this to npm as part of version 2.1.3.

**Comment by barchard** [comment]
Thanks. Are the failing CI checks okay?

**Comment by eelcocramer** [comment]
Yes. I had to drop CI support for ancient versions of nodejs (and iojs). Current master builds are ok.

File: src/windows/DeviceINQ.cc [hunk 1]
```diff
@@ -404,19 +404,75 @@ NAN_METHOD(DeviceINQ::ListPairedDevices) {
...
 
-                    Local<Array> servicesArray = Nan::New<v8::Array>((int)1);
-                    Local<Object> serviceObj = Nan::New<v8::Object>();
-
-                    serviceObj->Set(Nan::New("channel").ToLocalChecked(), Nan::New((int)1));
-                    serviceObj->Set(Nan::New("name").ToLocalChecked(), Nan::New("SPP").ToLocalChecked());
-                    servicesArray->Set(0, serviceObj);
+                    Local<Array> servicesArray = Local<Array>(Nan::New<Array>());
+                    {
+                        printf("enum service\n");
```
Comment (barchard) [inline src/windows/DeviceINQ.cc]: Should this printf be removed from production code?

Comment (eelcocramer) [inline src/windows/DeviceINQ.cc]: I guess so 👍

**Comment by checksummaster** [comment]
Sorry, my mistake, forgot to clean this one.
Also thank's for the release.

**Pull request merged**